### PR TITLE
add clear_config api

### PR DESCRIPTION
**Pull request by motobob**
Problem: 
#332 added function to add credentials into application config but no clear api was added 

Solution:
Add unset API for one config and all expired ones

### CODE REVIEW

**Comment by motobob** [comment]
@posilva here is the cleanup api added. please let me know if that matched your use case.

**Comment by posilva** [comment]
@motobob Yes, it ok for my use case. I would do it more conservative, with a bounded number (configurable) of allowed  cache entries, eventually with some LRU strategy. But it works for me.

**Comment by motobob** [comment]
@posilva I intentionally do not want to complicate the library with any caches and additional processes running. But it might change in future.

**Comment by motobob** [comment]
@kkuzmin any feedback?

**Comment by kkuzmin** [comment]
@motobob could you please also add cleanup call [here](https://github.com/erlcloud/erlcloud/blob/master/src/erlcloud_aws.erl#L606)

File: src/erlcloud_aws.erl [hunk 1]
```diff
@@ -411,6 +412,21 @@ update_config(#aws_config{} = Config) ->
                    security_token = Credentials#metadata_credentials.security_token}}
     end.
 
+-spec clear_config(aws_config()) -> ok.
+clear_config(#aws_config{assume_role = #aws_assume_role{role_arn = Arn, external_id = ExtId}}) ->
+    application:unset_env(erlcloud, {role_credentials, Arn, ExtId}).
+
+-spec clear_expired_configs() -> ok.
+clear_expired_configs() ->
```
Comment (kkuzmin) [inline src/erlcloud_aws.erl]: I'd rather name this function `clear_expired_credentials()`

**Comment by kkuzmin** [comment]
@motobob other than the comments above 👍 .

**Comment by motobob** [comment]
@kkuzmin 
>@motobob could you please also add cleanup call here

there is set_env call right at the end of that fun at https://github.com/erlcloud/erlcloud/blob/master/src/erlcloud_aws.erl#L632-L636

**Comment by kkuzmin** [comment]
@motobob thanks for the explanation. 👍

**Pull request merged**